docs/application/native/guides/ui/dali/capture.md

```diff
@@ -0,0 +1,66 @@
+# Capture
+
+
+The Dali::Capture is a class to capture a snapshot of the current scene. The captured result can be saved as a file and rendered by DALi `ImageView` or `ImageVisual`.
```

Comment:
1. Is it scene or screen? current screen?
2. Use code tag for `Dali::Capture`
3. Also, is that the file can be saved or can be rendered as source image? Is it either or condition, see below rephrase.

Please verify if this is better:

The `Dali::Capture` is a class to capture a snapshot of the current screen. The captured result can be saved as a file, or can be used as an image source to be rendered by DALi `ImageView` or `ImageVisual`.

Reply:
Thank you.
Actually, this api not capture the current screen. Exactly this capture a snapshot of a scene tree that consist of DALi controls.
So I used following sentence

The `Dali::Capture` is a class to capture a snapshot of the scene tree. The captured result can be saved as a file, or can be used as an image source to be rendered by DALi `ImageView` or `ImageVisual`.